pkg/ingester/ingester.go

```diff
@@ -268,7 +267,7 @@ func (i *Ingester) Flush(ctx context.Context, req *connect.Request[ingesterv1.Fl
 	i.instancesMtx.RLock()
 	defer i.instancesMtx.RUnlock()
 	for _, inst := range i.instances {
-		if err := inst.Flush(ctx); err != nil {
+		if err := inst.Flush(ctx, true, "api"); err != nil {
```

Comment:
Should `"api"` be a constant somewhere? Just like `flushReasonMaxBlockBytes` and `flushReasonMaxDuration`.

Reply:
may be ? It doesn't add much if it used once, but +1 on consistency.